tools/shoestring/tests/healthagents/test_rest_api.py

```diff
@@ -55,10 +55,11 @@ async def _process(_, response_body):
 
 def test_should_run_for_api_role():
 	# Act + Assert:
-	assert should_run(NodeConfiguration(NodeFeatures.API, *([None] * 6)))
+	assert should_run(NodeConfiguration(NodeFeatures.API, None, None, None, None, True, None, None))
```

Comment:
Should there be a ``should_run`` with False for the light node?
I know it should matter since it uses the ``NodeFeatures.API`` flag but just incase it change.

Reply:
you are right, I missed that.